src/firebase.js

```
@@ -0,0 +1,18 @@
+// src/firebase.js
+import { initializeApp } from "firebase/app";
+import { getMessaging, getToken, onMessage } from "firebase/messaging";
```

ESLint

[error] 'onMessage' is defined but never used.

(no-unused-vars)